.storybook/preview.js

```diff
@@ -0,0 +1,42 @@
+import 'emptykit.css'
+import * as React from 'react'
+import { withKnobs, boolean } from '@storybook/addon-knobs';
```

Comment:
extra semicolon

Reply:
thanks for spotting, fixed and all lint issues now resolved